src/ViewInstance.js

```diff
@@ -46,6 +52,15 @@ import {
 import { CalloutRenderer } from './components';
 
 import ImportRecordModal from './components/ImportRecordModal';
+import NewInstanceRequestBtn from './components/ViewInstance/MenuSection/NewInstanceRequestBtn';
```

Comment:
This is super petty, but just call it `...Button` instead of `...Btn`.

Reply:
Done.